lib/resque/tasks.rb

```diff
@@ -20,6 +20,15 @@
     worker.work(ENV['INTERVAL'] || 5) # interval, will block
   end
 
+  desc "Start a Resque worker"
+  task :work do
+    if ENV['PIDFILE'] && File.file?(ENV['PIDFILE'])
+      File.truncate(ENV['PIDFILE'], 0)
+    end
+    Rake::Task['resque:run'].reenable
+    Rake::Task['resque:run'].invoke
+  end
+
   desc "Start multiple Resque workers. Should only be used in dev mode."
```

Comment:
Do you understand the use case for this? Do you use this in production? Resque is designed around forking, so it seems strange that we support this. Also, shelling out to a rake task feels a bit dangerous to me for production-quality deployments.